packages/block-library/src/navigation/menu-items-to-blocks.js

```diff
@@ -21,7 +22,8 @@ export default function menuItemsToBlocks( menuItems ) {
 	}
 
 	const menuTree = createDataTree( menuItems );
-	return mapMenuItemsToBlocks( menuTree );
+	const blocks = mapMenuItemsToBlocks( menuTree );
+	return applyFilters( 'navigation.menuItemsToBlocks', blocks, menuItems );
```

Comment:
@adamziel @getdave if the filter would allow extenders to provide a drop in replacement to the default converter `mapMenuItemsToBlocks`, one starting from `menuTree`, instead of a way to change the already converted block tree, wouldn't that be a more generic solution?

Reply:
Then every implementation would need to access or re-implement `createDataTree` and `mapMenuItemsToBlocks`, plus it would be hard to stack multiple filters.